### ISSUE TITLE
Separate out GameThread issues

### ISSUE THREAD

**Issue by AaronGullickson** [AaronGullickson/mekhq#57]
I have some changes to GameThread in the story arcs PR that should really already be in MHQ separately. Notably, adding bots needs to be in the chat loung clause not oustide of it. Also, the check on deployment rounds. At the same time, the PR has some changes to AtBGameThread that it shouldn't so I should just replace that with the one in the master branch. However, I need to wait for Megamek/MekHQ#3972 to get addressed first.

**Comment by AaronGullickson** [comment]
deployRound on BotForce should be move out at the same time.

**Comment by AaronGullickson** [comment]
GameThread issue PR is merged. Need to deal with deployRound to knock this off the list but I might as well do all of the deployment parameters for BotForces.

**Comment by AaronGullickson** [comment]
I have put up Megamek/mekhq#3991 to address the deployment issue and I went ahead and put in all the new deployment variables there. Once that is merged into master and story-arcs, I should make adjustments to the young wolves arc where needed.

**Comment by AaronGullickson** [comment]
Its not merged and conflicts dealt with in de296a1. Two less files in the overall PR (from 88 to 86!).
